InfluxDB 2.0.4 (#2134)
influxdata/docs-v2: content/influxdb/v2.0/reference/config-options.md

---

### nats-max-payload-bytes
Maximum number of bytes allowed in a NATS message payload.

**Default:** `1048576`

| influxd flag               | Environment variable             | Configuration key        |
|:------------               |:--------------------             |:-----------------        |
| `--nats-max-payload-bytes` | `INFLUXD_NATS_MAX_PAYLOAD_BYTES` | `nats-max-payload-bytes` |

###### influxd flag
```sh
influxd --nats-max-payload-bytes=1048576
```

###### Environment variable
```sh
export INFLUXD_NATS_MAX_PAYLOAD_BYTES=1048576
```

###### Configuration file
{{< code-tabs-wrapper >}}
{{% code-tabs %}}
[YAML](#)
[TOML](#)
[JSON](#)
{{% /code-tabs %}}
{{% code-tab-content %}}
```yml
nats-max-payload-bytes: 1048576
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```toml
nats-max-payload-bytes = 1048576
```
{{% /code-tab-content %}}
{{% code-tab-content %}}
```json
{
  "nats-max-payload-bytes": 1048576
}
```
{{% /code-tab-content %}}
{{< /code-tabs-wrapper >}}

---

### nats-port

> Port to be bound by the NATS streaming server. `-1` selects a random port.

Reviewer: to bind/connect to?

> Port to be bound by the NATS streaming server. `-1` selects a random port.

Reviewer 2: ```suggestion
Port for the NATS streaming server. `-1` selects a random port.
```
